### PR TITLE
lang/gcc48,49,5,6: Refine the condition to enable multilib

### PR DESCRIPTION
**Pull request by bartoszkosiorek**
It fixes the build issue on Ubuntu 20.04

Pull Request created after comments from:
https://github.com/NetBSD/pkgsrc/pull/62

### CODE REVIEW

**Comment by krytarowski** [comment]
Please try to push these paths to: mk/platform/Linux.mk

**Comment by krytarowski** [comment]
Ah, done I was apparently looking at an older revision, checking.

File: mk/platform/Linux.mk [hunk 1]
```diff
@@ -49,6 +49,24 @@ _OPSYS_EMULDIR.linux=	# empty
...
 
 # Support Debian/Ubuntu's multiarch hierarchy.
+
+###
+### Determine if multilib is available.
+###
+MULTILIB_SUPPORTED?=	unknown
+.if !empty(MACHINE_PLATFORM:MLinux-*-x86_64)
+.  if exists(/usr/include/x86_64-linux-gnu/gnu)
+_GNU_INCLUDE_DIR=	/usr/include/x86_64-linux-gnu/gnu
```
Comment (krytarowski) [inline mk/platform/Linux.mk]: You should add something like `_OPSYS_INC_DIR`, on par with `_OPSYS_LIB_DIRS` and handle similarly.

**Comment by krytarowski** [comment]
@jperkin could you please handle this as you can have access to Linux and other platforms?

File: mk/platform/Linux.mk (hunk 1 above)
Comment (bartoszkosiorek) [reply mk/platform/Linux.mk]: I have renamed to `_OPSYS_GNU_INCLUDE_DIR` on par with `_OPSYS_INCLUDE_DIRS`

**Comment by bartoszkosiorek** [comment]
@krytarowski What else needs to be done to accept this Pull Request?

**Comment by krytarowski** [comment]
@bartoszkosiorek-tomtom I will leave you a mail, please contact me on kamil@netbsd.org

**Pull request closed without merging**